### ISSUE TITLE
won't validate KML files

### ISSUE THREAD

**Issue by ashtrax3000** [sissaschool/xmlschema#148]
It won't work in python3 when validating KML files using ogckml23.xsd (XML schema 1.1) and also setting class to XMLSchema11

http://schemas.opengis.net/kml/2.3/ogckml23.xsd

**Comment by brunato** [comment]
Hi,

it's an error on schema parsing (a check that have to be relaxed in XSD 1.1):

```python
>>> xs = xmlschema.XMLSchema11("test_cases/opengis/kml/2.3/ogckml23.xsd", validation='lax')
>>> xs.all_errors[-1]
XMLSchemaParseError(XsdAttributeGroup(['hint', 'version', Xsd11AnyAttribute(namespace=['##other'], process_contents='lax')]), 'another declaration after anyAttribute', <Element '{http://www.w3.org/2001/XMLSchema}complexType' at 0x7ff6a49dc770>)
```
also there is an additional error related into a wrong check on an abstract element that belong to a substitution group.
The fix for both errors will be available with the next release.
Thanks

**Comment by brunato** [comment]
Fix available with v1.0.17+.
Bye
